setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+setup(
+    name = 'bayesopt',
+    version = '0.1',
+    #packages = find_packages(),
+    packages = ["BayesOpt", "BayesOpt.optimizer"],
+    #Metadata
+    author = 'wangronin',
```

Comment:
Thanks! However, I think it won't be fair because I just added a few lines to your code... Once you merge the branches I will appear in the contributors' section.